AppCenterCrashes/AppCenterCrashesTests/MSErrorLogFormatterTests.mm

```diff
@@ -164,12 +164,15 @@ - (void)testProcessIdAndExceptionForObjectiveCExceptionCrash {
 }
 
 - (void)testSelectorForRegisterWithName {
-  NSData *crashData = [[[MSCrashes sharedInstance] plCrashReporter] generateLiveReport];
+  
```

Comment:
Does switching this to a fixture help with improving the flakiness of this test? This is interesting!

Reply:
On the other hand, the `generateLiveReport`-API doesn't generate a "full" report but a "live crash report" that's non-crashing, so probably not the way to unit test the selectors anyway. ^^

Reply:
I've faced that `generateLiveReport` may return `nil`, and it happens sometimes on CI.

Reply:
ok